tests/tracing/test_sampling.py

```diff
@@ -310,7 +336,10 @@ def test_records_lost_event_only_if_traces_sampler_enabled(
     sampled_output,
     expected_record_lost_event_calls,
 ):
-    sentry_init(traces_sampler=traces_sampler)
+    sentry_init(
+        traces_sample_rate=None,
```

Comment:
This test passes only if `traces_sample_rate=None` and it fails when it is `0` or `1.0`. I am not sure if this is the excepted behavior.